src/common/SPELLS/priest.ts

```diff
@@ -491,7 +474,62 @@ const spells = spellIndexableList({
     name: 'Binding Heals',
     icon: 'spell_holy_blindingheal',
   },
-
+  HOLY_WORD_SALVATION_TALENT: {
+    id: 265202,
+    name: 'Holy Word: Salvation',
+    icon: 'ability_priest_archangel',
+    manaCost: 3000,
+  },
+  APOTHEOSIS_TALENT: {
+    id: 200183,
+    name: 'Apotheosis',
+    icon: 'ability_priest_ascension',
+  },
+  DIVINE_WORD_TALENT: {
+    id: 372760,
+    name: 'Divine Word',
+    icon: 'spell_priest_chakra',
+  },
+  LIGHTWELL_TALENT: {
+    id: 372835,
+    name: 'Lightwell',
+    icon: 'spell_holy_summonlightwell',
+    manaCost: 1500,
+  },
+  SYMBOL_OF_HOPE_TALENT: {
+    id: 64901,
+    name: 'Symbol of Hope',
+    icon: 'spell_holy_symbolofhope',
+  },
+  CIRCLE_OF_HEALING_TALENT: {
+    id: 204883,
+    name: 'Circle of Healing',
+    icon: 'spell_holy_circleofrenewal',
+    manaCost: 1650,
+  },
+  HOLY_WORD_CHASTISE_TALENT: {
+    id: 88625,
+    name: 'Holy Word: Chastise',
+    icon: 'spell_holy_chastise',
+    manaCost: 1000,
+  },
+  HOLY_WORD_CHASTISE_CENSURE_TALENT_INCAPACITATE: {
+    id: 200196,
+    name: 'Holy Word: Chastise',
+    icon: 'spell_holy_chastise',
+    manaCost: 1000,
+  },
+  HOLY_WORD_CHASTISE_CENSURE_TALENT_STUN: {
+    id: 200200,
+    name: 'Holy Word: Chastise',
+    icon: 'spell_holy_chastise',
+    manaCost: 1000,
+  },
```

Comment:
these spells won't be pulled in by auto-generated talents, so they're fine to include here